pkgs/dart_services/lib/src/analysis.dart

```diff
@@ -354,10 +354,6 @@ class AnalysisServerWrapper {
         ...importIssues,
         ...issues,
       ],
-      packageImports: imports
```

Comment:
I'd continue to populate the field (but would make the field nullable at the definition).